src/translations/i18n.js

```diff
@@ -6,35 +6,42 @@ import {
   enUSIndividualScoreReport,
   esTranslations,
   esCOTranslations,
+  deTranslations,
   esIndividualScoreReport,
   esCOIndividualScoreReport,
 } from './exports';
+import { isLevante } from '@/helpers';
 
 export const languageOptions = {
   'en-US': { translations: enUSTranslations, language: 'English (United States)', code: 'usa' },
   en: { translations: enTranslations, language: 'English (United Kingdom)', code: 'gb' },
   es: { translations: esTranslations, language: 'Español (Spain)', code: 'es' },
   'es-CO': { translations: esCOTranslations, language: 'Español (América Latina)', code: 'col' },
+  de: { translations: deTranslations, language: 'Deutsch', code: 'de' },
 };
-export let browserLocale = window.navigator.language;
 
-const getLocale = (locale) => {
-  if (Object.keys(languageOptions).includes(locale)) {
-    console.log('Locale found in languageOptions: ', locale);
-    return locale;
-  } else if (locale.includes('es')) {
-    console.log('Spanish dialect not supported, using default es.');
-    return 'es';
+const browserLocale = window.navigator.language;
+
+const getLocale = (localeFromBrowser) => {
+  const localeFromStorage = sessionStorage.getItem(`${isLevante ? 'levante' : 'roar'}PlatformLocale`);
+
+  if (localeFromStorage) {
+    return localeFromStorage;
   } else {
-    console.log('Language not supported, using default en-US.');
-    return 'en-US';
+    sessionStorage.setItem(`${isLevante ? 'levante' : 'roar'}PlatformLocale`, localeFromBrowser);
+    return localeFromBrowser;
   }
 };
```

Comment:
Now that I got to that part in the PR, I understand better what you're implementing with the session storage in the language switcher. 

In that case, why do we need to differentiate between ROAR and LEVANTE in the locale storage? I'd say we can just store this a simple key for both platforms? Either way, let's use the constants mentioned earlier please.

Also: why store the preference in session storage? Is this purely intended for manual window reloads? If the user closes the tab/window and opens the dashboard again, the language preference will be gone.